riak/codecs/ttb.py

```diff
@@ -154,7 +154,10 @@ def decode_timeseries(self, resp_ttb, tsobj,
         self.maybe_err_ttb(resp_ttb)
 
         resp_a = resp_ttb[0]
-        if resp_a == tsputresp_a:
+
+        if resp_ttb == tsqueryresp_a:
```

Comment:
OK,  I would argue the bug is in `riak_kv`, and the returned data should be `{tsqueryresp, {[], [], []}}`, but I guess this is simpler. I will re-work this since I misread that you were comparing the *entire* ttb message as the atom after the statement that extracts the first element from (what I expected to be) a tuple.